rhea/build/toolflow/xilinx/vivado.py

```diff
@@ -8,6 +8,7 @@
 
 import sys
 import os
+import platform
```

Comment:
Platform is python standard library ( https://docs.python.org/3/library/platform.html ).   Confusingly I didn't use it in this changelist, but I do use it in a future changelist in this pull request to add '.bat'.